### PR TITLE
Update product tutorial

### PR DESCRIPTION
**Pull request by marcoles**
This sample is a bit different than the python one because here we don't use the updateMask field

### CODE REVIEW

File: tutorials/retail_api_v2_update_product_java.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  <meta name="description" content="Learn how to use Retail API Java library to update product tutorial" />
+  <meta name="component_id" content="593554" />
+  <meta name="keywords" content="retail, update product" />
+</walkthrough-metadata>
+
+# Update product tutorial
+
+## Get started
+
+To fill the catalog or to update a massive number of products, we recommend using the `import_products` method. However,
```
Comment (pmichalskiGD) [inline tutorials/retail_api_v2_update_product_java.md:12]: let's remove all _ in method and field names, they are not used in Java for non final fields and method names

File: tutorials/retail_api_v2_update_product_java.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+1. Go to the code samples directory - our starting point to run more commands.
+    ```bash
+    cd java-retail/samples/snippets
+    ```
+
+## Update product
+
+To update a product you should send an `UpdateProductRequest` call to the Retail API with the following required fields specified:
+- `product`—the product object to be updated or created (depending on the  `allow_missing` value, the product can be created if it's missing).
+- `allow_missing`—if the value is set to `true`, and the product is not found, a new product is created.
```
Comment (pmichalskiGD) [inline tutorials/retail_api_v2_update_product_java.md:134]: same here, please remove _

**Review by pmichalskiGD — APPROVED** [review]

File: tutorials/retail_api_v2_update_product_java.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+1. Before you start, build the Maven project and go to the code samples directory - our starting point to runcode samples:
```
Comment (DawidJeziorskiGrid) [inline tutorials/retail_api_v2_update_product_java.md:136]: Is:
our starting point to runcode samples:
maybe it should be:
our starting point to run code samples:

**Review by DawidJeziorskiGrid — APPROVED** [review]
Small error from copied materials.

File: tutorials/retail_api_v2_update_product_java.md (hunk 1 above)
Comment (marcoles) [reply tutorials/retail_api_v2_update_product_java.md]: done

File: tutorials/retail_api_v2_update_product_java.md (hunk 2 above)
Comment (marcoles) [reply tutorials/retail_api_v2_update_product_java.md]: done

File: tutorials/retail_api_v2_update_product_java.md (hunk 3 above)
Comment (marcoles) [reply tutorials/retail_api_v2_update_product_java.md]: fixed

**Pull request merged**